source/index.ts

```diff
@@ -30,6 +30,7 @@ export interface Options {
 	readonly css?: string;
 	readonly script?: string;
 	readonly cookies?: ReadonlyArray<string | Record<string, string>>;
+	readonly launchOptions?: Record<string, string[]>;
```

Comment:
This should be typed like it's done in `capture-websites`.

Reply:
It seems that in `capture-websites`, `LaunchOptions` is imported from puppeteer in order to state:

`readonly launchOptions?: LaunchOptions;`

As such, should puppeteer be added to pageres's `package.json` to enable its import?

Reply:
You can just get the type from the `captureWebsites.Options.launchOptions` type or something.

Reply:
I believe I have made the necessary changes to match your requirements. 
For some reason, the pipeline for `Node.js 10` seems to be taking a long time though (as it happened yesterday when I changed a line in the readme).